test/test_testing.py

```diff
@@ -136,8 +140,10 @@ def test_can_connect(self):
         harness.begin()
         c = harness.model.unit.get_container('foo')
         self.assertTrue(c.can_connect())
+        ops.testing.SIMULATE_CAN_CONNECT = tmp
 
     def test_simulate_can_connect(self):
+        tmp = ops.testing.SIMULATE_CAN_CONNECT
```

Comment:
you should really have a self.addCleanup here rather than just setting it at the end.
because if the test fails, then the value is in an indeterminate state.

Reply:
done